browser/resources/settings/brave_sync_page/brave_sync_subpage.ts

```diff
@@ -70,10 +71,19 @@ class SettingBraveSyncSubpage extends SettingBraveSyncSubpageBase {
       },
 
       /** @private */
-      hasSyncWordsDecryptionError_:{
+      hasLockedSafeStorage_ : {
         type: Boolean,
         value: false,
-        computed: 'computeHasSyncWordsDecryptionError_(syncStatus.hasSyncWordsDecryptionError)',
+        computed: 'computeLockedSafeStorage_(' +
+          'syncStatus.hasSyncWordsDecryptionError)',
+      },
+
+      /** @private */
+      hasUnlockedSafeStorageCannotDecryptSeed_ : {
+        type: Boolean,
+        value: false,
+        computed: 'computeUnlockedSafeStorageCannotDecryptSeed_(' +
```

Comment:
It depends on all of syncStatus, so just pass that through instead of only 1 property from it.

Reply:
thanks, fixed at 
https://github.com/brave/brave-core/pull/16117/commits/a1c36df05d3828a26dd556fb9a84f0a660de268b